test/spec/modules/eids_spec.js

```diff
@@ -238,6 +238,40 @@ describe('eids array generation for known sub-modules', function() {
     });
   });
 
+
+  it('sovrn', function() {
+    const userId = {
+      sovrn: {'id': 'sample_id'}
+    };
+    const newEids = createEidsArray(userId);
+    expect(newEids.length).to.equal(1);
+    expect(newEids[0]).to.deep.equal({
+      source: 'liveintent.sovrn.com',
+      uids: [{
+        id: 'sample_id',
+        atype: 3
+      }]
+    });
+  });
+
+  it('sovrn with ext', function() {
+    const userId = {
+      sovrn: {'id': 'sample_id', 'ext': {'provider': 'some.provider.com'}}
+    };
+    const newEids = createEidsArray(userId);
+    expect(newEids.length).to.equal(1);
+    expect(newEids[0]).to.deep.equal({
+      source: 'media.net',
```

Comment:
```suggestion
      source: 'liveintent.sovrn.com',
```

Reply:
Good catch. Some copy pasta. But the test run was green. Hmm. 

Fixed this one and the one in the ext below in https://github.com/LiveIntent/Prebid.js/pull/26/commits/664636e26c9700c853d2ff14b507dc506202dfce.

Reply:
Its js magic